src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/illegalidentifiername/Example1.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="IllegalIdentifierName"/>
+  </module>
+</module>
+*/
+
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.naming.illegalidentifiername;
+
+// xdoc section -- start
+public class Example1 {
+  public static void main(String... args) {
+    var var = 4; // violation
```

Comment:
Please do us a favor, remove method and let of of variables become a class fields.

It will make example shorter horizontally.

Reply:
done